data/json/vehicleparts/utilities.json

```
@@ -169,6 +169,40 @@
     "symbol": "&",
     "type": "vehicle_part"
   },
+  {
+    "abstract": "adv_butchery_abstract",
+    "breaks_into": [
+      { "count": [ 4, 6 ], "item": "steel_lump" },
+      { "count": [ 4, 6 ], "item": "steel_chunk" },
+      { "count": [ 4, 6 ], "item": "scrap" },
+      { "item": "pan", "prob": 50 },
+      { "item": "pot", "prob": 50 }
+    ],
+    "broken_color": "light_cyan",
+    "broken_symbol": "x",
+    "color": "light_cyan",
+    "damage_modifier": 10,
+    "damage_reduction": { "all": 30 },
+    "description": "Combining a fixed-up dissector, workbench and advanced electronics, this is a marvel capable of butchering, dissecting and cutting like nothing else!",
+    "durability": 80,
+    "flags": [ "CARGO", "OBSTACLE", "BUTCHER_EQ", "TRANSPARENT", "FLAT_SURF" ],
+    "item": "adv_butchery",
+    "location": "center",
+    "looks_like": "f_scan_bed",
```

If I recall, vehiclepart looks-like can only make it look like other vehicleparts, we need to either make support for looking like furniture work someday or just convert all the other broken looks-like entries to point to other VPs...

As for this one, I guess vehicle autodoc is close enough appearance-wise to work hopefully?
```suggestion
    "looks_like": "vp_autodoc",
```

---

I could try adding another id into tiles_configs for current tileset that come to the game however it's beyond the scope of this PR currently, this will works